stripe/src/main/java/com/stripe/android/view/PaymentFlowActivity.java

```diff
@@ -15,13 +15,13 @@
 import java.util.List;
 
 /**
- * Activity containing a two-part shipping flow that allows users to provide a shipping address
+ * Activity containing a two-part payment flow that allows users to provide a shipping address
  * as well as select a shipping method.
  */
-public class ShippingFlowActivity extends StripeActivity {
+public class PaymentFlowActivity extends StripeActivity {
 
     private ViewPager mViewPager;
-    private ShippingFlowPagerAdapter mShippingFlowPagerAdapter;
+    private PaymentFlowPagerAdapter mPaymentFlowPagerAdapter;
 
     static final String EXTRA_SHIPPING_FLOW_CONFIG = "shipping_flow_config";
```

Comment:
probably also worth a rename? Or are you keeping the shipping flow separate from the payment flow config?

Reply:
oh yes yes, oversight here.